<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WordCounter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserire URL del file di testo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditURL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDownload">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDownloadedData">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>531</width>
      <height>501</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>40</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelParole">
       <property name="text">
        <string>numero di parole</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaratteri">
       <property name="text">
        <string>numero di caratteri</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaratteri2">
       <property name="text">
        <string>numero di caratteri senza spazi:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrasi">
       <property name="text">
        <string>numero di frasi:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelParagrafi">
       <property name="text">
        <string>numero di paragrafi:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
